app/views/publish/providers/schools/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= content_for :page_title, @site.location_name %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_recruitment_cycle_schools_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h1 class="govuk-fieldset__heading govuk-!-margin-bottom-3"><%= @site.location_name %></h1>
+      </legend>
+        <%= render GovukComponent::SummaryListComponent.new do |component|
+              component.with_row do |row|
+                row.with_key { t("publish.providers.schools.location_name") }
+                row.with_value(text:  @site.location_name)
```

Comment:
This needs to be `School name` as the UI display

Reply:
corrected